### DISCUSSION TITLE
speculation rules

### DISCUSSION THREAD

**Discussion by danielciornei** [breezefront/community#277] [1. Help]
was playing round with https://github.com/collabpl/magento2-module-speculation-rules 

any plans to implement it in breeze? 
if not how to allow this type of scripts on breeze? 

<script type="speculationrules">
{
  "prerender": [{
    "where": {
      "and": [
        { "href_matches": "/*" },
        { "not": {"selector_matches": ".do-not-prerender"}},
        { "not": {"selector_matches": "[rel=nofollow]"}}
      ]
    }
  }]
}
</script>

**Comment by vovayatsyuk** [comment]
1. Yes, I think we will add it at some point.
2. Breeze should not remove this script. Are you sure you use browser with `speculationrule` support? This module adds rules only if browser supports them: https://github.com/collabpl/magento2-module-speculation-rules/blob/main/view/frontend/templates/script.phtml#L26
